ts/Core/Time.ts

```diff
@@ -410,6 +410,7 @@ class Time {
             return (timestamp: number | Date): number => {
 
                 try {
+                    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
This is a case when there are unused vars before the ones that are used. Can we use the [after-used](https://eslint.org/docs/latest/rules/no-unused-vars#:~:text=for%20variables%20and-,after%2Dused,-for%20arguments.) flag so that we don't need an exception for this case?

Reply:
hmm, the "" rule doesn't support support after-used on deconstructions as far as I could find, so proposing a minor refactor to:
```
const [hours, minutes = 0] =
    // eslint-disable-next-line new-cap
    Intl.DateTimeFormat('en', {
        timeZone: options.timezone,
        timeZoneName: 'shortOffset'
    })
        .format(timestamp)
        .split(/(GMT|:)/)
        .map(Number)
        // Filter NaNs (string values like "GMT" and ":")
        .filter(function (predicate): boolean {
            return !isNaN(predicate);
        }),
    offset = -(hours + minutes / 60) * 60 * 60000;
```